agent/ansible/pbench/agent/README.md

```diff
@@ -0,0 +1,94 @@
+# Ansible Collection - pbench.agent
+
+The roles in this collection allow the installation of the pbench-agent RPM and related RPMs, and the configuration of the agent for use in a particular environment.
+
+More information about pbench can be found at https://github.com/distributed-system-analysis/pbench.
+
+The roles assume that the following variables are defined:
+
+- pbench_repo_url_prefix: the repo where the RPMs can be found (default: https://copr-be.cloud.fedoraproject.org/results/ndokos).
+- pbench_config_url: the location where one or more config files can be obtained (default: none).
+- pbench_config_dest: the location within the pbench-agent installation directory where the config file is going to be copied (default: /opt/pbench-agent/config/)
+- pbench_config_files: a list of config files to be installed on an agent (default: '["pbench-agent.cfg"]').
+- pbench_key_url: the location where an ssh key can be obtained for sending results to a server (default: none).
+- pbench_key_dest: the location within the pbench-agent installation directory where the key is going to be copied. (default: /opt/pbench-agent/).
+
+Some of these are defaulted for convenience, but pbench_repo_url_prefix and pbench_key_url are specific to an environment and need to be specified locally (most likely in the inventory file).
+
+Here's an example playbook:
+
+```
+---
+- name: install pbench-agent
+  hosts: servers
+  become: yes
+  become_user: root
+
+  # The default value ('production') can be overriddent by cenv, a host-specific
+  # inventory variable.
+  vars:
+    pbench_configuration_environment: "{{ cenv | default('production') }}"
+
+  roles:
+    - pbench_repo_install
+    - pbench_agent_install
+    - pbench_agent_config
+```
+
+and an example inventory file that can be edited to suit your circumstances:
+
+```
```

Comment:
Could we keep an example file in the git repo and remove it from this `README.md` file?

Reply:
See below.